tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -44,7 +54,14 @@ namespace {
 class RUInt16BE {
 private:
    std::uint16_t fValBE = 0;
-   static std::uint16_t Swap(std::uint16_t val) { return (val & 0x00FF) << 8 | (val & 0xFF00) >> 8; }
+   static std::uint16_t Swap(std::uint16_t val)
+   {
+#if R__LITTLE_ENDIAN == 1
+      return RByteSwap<sizeof(fValBE)>::bswap(val);
```

Comment:
Maybe use `sizeof(val)` instead of the member `fValBE` since we are in a `static` function.